Update panel improvements.

3Dgoo opened the pull request:
Improving how the update panel compares version numbers.

3Dgoo committed:
Update panel improvements.

Turnerj commented on code/panels/UpdatePanel.php:
```diff
@@ -83,28 +85,42 @@ public function getLatestSilverStripeVersion() {
-		if ($this->getCurrentSilverStripeVersion() && $this->getLatestSilverStripeVersion()) {
-			return $this->getCurrentSilverStripeVersion() >= $this->getLatestSilverStripeVersion();
+	public function isNewerSilverStripeVersion($versionNumber, $latestVersionNumber) {
+		if ($versionNumber == $latestVersionNumber) {
```
Not so sure on this part. I get why you have it but it conflicts with the name of the function. Maybe the function needs to be reversed so it is `isOlderSilverStripeVersion`?

3Dgoo replied:
Or `isNewestSilverStripeVersion`? What do you think?

3Dgoo replied:
Or I could remove that if check completely. But I think it's nice to have in there because it saves the effort of comparing every part of the string for no reason.

Turnerj replied:
Yeah, maybe `isNewestSilverStripeVersion` makes more sense - it checks whether the first value is the newest or the same as the second value.

I agree about keeping the logic there, it is a nice check-and-skip case.

3Dgoo committed:
Changing update panel function name

Changing `isNewerSilverStripeVersion` function name to `isNewestSilverStripeVersion` in update panel code.

3Dgoo commented:
I have changed the function name to `isNewestSilverStripeVersion` as suggested.
